.github/workflows/windows.yml

```diff
@@ -0,0 +1,26 @@
+name: windows
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  perl:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        perl-version:
+          - '5.30'
+          - '5.32'
+    name: Perl ${{ matrix.perl-version }} on Windows
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up perl
+        uses: shogo82148/actions-setup-perl@v1
+        with:
+          perl-version: ${{ matrix.perl }}
```

Comment:
Isn't it matrix.perl-version?

Reply:
Thanks, fixed https://github.com/tokuhirom/Test-TCP/pull/99/commits/b87625ef3cff52c6e3e917ea9e889bfe7d19a4ab